src/test/java/com/znsio/workflow/jsonPlaceHolderTemplate.feature

```diff
@@ -0,0 +1,24 @@
+  @posts @prod
+  Feature: Api workflow tests for Json Place Holder
+
+    Background:
+      * def userId = generateRandomNumber(3)
+      * def title = generateAlphaNumericRandomString(5)
+      * def body = generateAlphaNumericRandomString(10)
+      * def jsonPayload = { "userId": "#(userId)", "title": "#(title)", "body": "#(body)" }
+      * karate.log("jsonPayload : "+jsonPayload.userId+" "+jsonPayload.title+" "+jsonPayload.body)
+
+    Scenario: User should be able to create a post and then edit the created post
+      Given def postedUserResponse = karate.call('classpath:com/znsio/templates/postsTemplates.feature@t_postPostsForUser', {'jsonPayload' : jsonPayload , 'expectedStatus': 201}).postForUser
```

Comment:
This is wrong !. First verify the Response of fist API call then if its working ok then call another API.